src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```diff
@@ -38,15 +38,20 @@
 
 
 def set_feature(cmd,
-                feature,
+                feature=None,
+                key=None,
                 name=None,
                 label=None,
                 description=None,
                 yes=False,
                 connection_string=None,
                 auth_mode="key",
                 endpoint=None):
-    key = FeatureFlagConstants.FEATURE_FLAG_PREFIX + feature
+    if key is None and feature is None:
+        raise CLIError("Please provide either `--key` or `--feature` value.")
```

Comment:
Could you please use the specific error type `RequiredArgumentMissingError` instead of `CLIError`